openassessment/xblock/leaderboard_mixin.py

```diff
@@ -72,13 +75,16 @@ def render_leaderboard_complete(self, student_item_dict):
         for score in scores:
             if 'file_key' in score['content']:
                 score['file'] = file_upload_api.get_download_url(score['content']['file_key'])
-            if 'text' in score['content']:
-                score['content'] = score['content']['text']
+            if 'text' in score['content'] or 'parts' in score['content']:
```

Comment:
Actually, most of the backwards compatibility support is in `data_conversion.py`. It looks like I missed this case.